LiteNetLib/BaseChannel.cs

```diff
@@ -16,7 +16,13 @@ protected BaseChannel(NetPeer peer)
 
         public int PacketsInQueue
         {
-            get { return OutgoingQueue.Count; }
```

Comment:
This was intentional. Because this count doesn't used to access queue. Just for info.